tutorial_R/summarySE.R

```diff
@@ -21,20 +20,20 @@ summarySE <- function(data=NULL, measurevar, groupvars=NULL, na.rm=FALSE,
   
   # This does the summary. For each group's data frame, return a vector with
   # N, mean, median, and sd
-  datac <- ddply(data, groupvars, .drop=.drop,
-                 .fun = function(xx, col) {
-                   c(N    = length2(xx[[col]], na.rm=na.rm),
-                     mean = mean   (xx[[col]], na.rm=na.rm),
-                     median = median (xx[[col]], na.rm=na.rm),
-                     sd   = sd     (xx[[col]], na.rm=na.rm)
-                   )
-                 },
-                 measurevar
+  datac <- plyr::ddply(data, groupvars, .drop=.drop,
+                       .fun = function(xx, col) {
+                           c(N    = length2(xx[[col]], na.rm=na.rm),
+                             mean = mean   (xx[[col]], na.rm=na.rm),
+                             median = median (xx[[col]], na.rm=na.rm),
+                             sd   = sd     (xx[[col]], na.rm=na.rm)
+                           )
+                       },
+                       measurevar
```

Comment:
What does this change do? Does it functionally make a difference to the code? (Sorry I'm not an R person so I can't review very well!)

Reply:
Ah, I didn't want to load plyr into the namespace (there are some collisions with `rename()` in `summarySE.R`, which is an issue depending on the order in which packages are loaded).
So I made all the calls to plyr functions explicit and then dropped the code to load it in.

Reply:
Makes sense to me! ✅